File: CT_BAEKJOON_14681.py
#https://www.acmicpc.net/problem/14681
#https://www.acmicpc.net/source/41979354

a = input()
b = input()
a = int(a)
b = int(b)


if a>0 and b > 0:
    print(1)
elif a>0 and b <0:
    print(4)
elif a<0 and b<0:
    print(3)
elif a<0 and b > 0:
    print(2)

File: CT_BAEKJOON_14681.c
//https://www.acmicpc.net/problem/14681
//https://www.acmicpc.net/source/40066916

#include <stdio.h>
#pragma warning(disable:4996)
int main()
{
	int A , B , fir ,sec, th;
	scanf("%d %d", &A , &B);
	if (A > 0){

		if (B >0) {
			printf("1");
		}
		else {
			printf("4");
		}

	}
	else {
		if (B > 0) {
			printf("2");
		}
		else {
			printf("3");
		}
	}
}	